.golangci.yml

```
@@ -12,7 +12,7 @@ run:
 # output configuration options
 output:
   # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
-  format: tab
+  formats: tab
```

sure this is correct? maybe
```suggestion
  formats:
    - format: tab
```
https://golangci-lint.run/usage/configuration/

---

You are right, fixed